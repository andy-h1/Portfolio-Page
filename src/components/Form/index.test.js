import React from 'react';
import { render } from '@testing-library/react';
import { Form } from '.';

describe('Form Component', () => {
  it('should render', () => {
    const { container } = render(<Form />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
