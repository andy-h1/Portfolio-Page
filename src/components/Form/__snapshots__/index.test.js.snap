// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component should render 1`] = `
<div
  class="sc-AxmLO jPFMJW"
>
  <h2
    class="sc-fzozJi fvRlLu"
  >
    Sign Up Form
  </h2>
  <form
    class="sc-fzoLsD gxuKiv"
  >
    <input
      class="sc-fzpans idaupV"
      id="name"
      placeholder="Username"
      required=""
      type="text"
      value=""
    />
    <input
      class="sc-fzpans idaupV"
      id="email"
      placeholder="Email Address"
      required=""
      type="email"
      value=""
    />
    <input
      class="sc-fzpans idaupV"
      id="age"
      max="60"
      min="18"
      placeholder="Age"
      type="number"
      value=""
    />
    <div
      class="sc-fznyAO XyPLH"
    >
      <input
        class="sc-fzpans idaupV"
        id="password"
        placeholder="Password"
        required=""
        type="password"
        value=""
      />
      <button
        class="sc-fznZeY YWaki"
        type="button"
      />
    </div>
    <button
      class="sc-fzplWN fViOlb"
      type="submit"
    >
      Register
    </button>
  </form>
  <div
    class="sc-fznKkj erwKkS"
  />
</div>
`;
